component/job_status.go

```diff
@@ -0,0 +1,48 @@
+package component
+
+import (
+	"github.com/rivo/tview"
+	primitive "github.com/hcjulz/damon/primitives"
+)
+
+type JobStatus struct {
+	TextView TextView
+	Status    string
```

Comment:
Would you mind to add another struct `JobStatusProps` at this place and make the status part of that struct? Just to make it consistent with the other components (you can compare it for example to `JobsTable`).

I will add another readme with some implementation details on how a component is structured. But usually what I do is to add a `Props` field to the component struct. Props usually contain dynamic variables (variables that change and the render function reads from to render the information). Does that make sense?

Reply:
Created separate model to hold the JobStatus data.